packages/angular_devkit/core/src/json/schema/registry.ts

```diff
@@ -223,29 +342,33 @@ export class CoreSchemaRegistry implements SchemaRegistry {
 
     return validator
       .pipe(
-        map<ajv.ValidateFunction, SchemaValidator>(validate => (data, options) => {
-          const validationOptions: SchemaValidatorOptions = {
-            withPrompts: true,
-            ...options,
-          };
```

Comment:
This was used to make defaults more obvious and eliminated the need to check for undefined everywhere the option was used if the default was true.  It was the reason Partial<> was used in the function declaration as well.

Reply:
Removing that code did not change anything else. You're checking that `withPrompt === false` to shortcut the code.

Reply:
Reason I mentioned it is that there are two code blocks similar to the following farther down:
```
if (options === undefined
	      || options.applyPreTransforms === undefined
              || options.applyPreTransforms) {
```

Reply:
Done.